iam.tf

```diff
@@ -126,6 +126,16 @@ data "aws_iam_policy_document" "ssm_s3_cwl_access" {
 
     resources = [aws_kms_key.ssmkey.arn]
   }
+
+  statement {
+    sid = "SecretsManagerAccessForSessionManager"
+
+    actions = [
+      "secretsmanager:GetSecretValue"
+    ]
+
+    resources = ["arn:aws:secretsmanager:*:*:secret:control/*"]
```

Comment:
This seems quite a control specific thing to put into this module

Reply:
I tried to restrict it as much as possible so that I don't give too much access to the resources

Reply:
I see that but it does mean this is no longer a reusable module